docs/src/main/paradox/orientdb.md

```diff
@@ -0,0 +1,124 @@
+
+Scala
+: @@snip ($alpakka$/orientdb/src/test/scala/akka/stream/alpakka/orientdb/OrientDBSpec.scala) { #init-settings }
+
+Java
+: @@snip ($alpakka$/orientdb/src/test/java/akka/stream/alpakka/orientdb/OrientDBTest.java) { #init-settings }
+
+We will also need an @scaladoc[ActorSystem](akka.actor.ActorSystem) and an @scaladoc[ActorMaterializer](akka.stream.ActorMaterializer).
+
+Scala
+: @@snip ($alpakka$/orientdb/src/test/scala/akka/stream/alpakka/orientdb/OrientDBSpec.scala) { #init-mat }
+
+Java
+: @@snip ($alpakka$/orientdb/src/test/java/akka/stream/alpakka/orientdb/OrientDBTest.java) { #init-mat }
+
+This is all preparation that we are going to need.
+
+## ODocument message
+
+Now we can stream messages which contains OrientDB's `ODocument` (in Scala or Java)
+from or to OrientDB where we have access to by providing the `ODatabaseDocumentTx` to the
+@scaladoc[OrientDBSource](akka.stream.alpakka.orientdb.scaladsl.OrientDBSource$) or the
+@scaladoc[OrientDBSink](akka.stream.alpakka.orientdb.scaladsl.OrientDBSink$).
+
+Scala
+: @@snip ($alpakka$/orientdb/src/test/scala/akka/stream/alpakka/orientdb/OrientDBSpec.scala) { #run-odocument }
+
+Java
+: @@snip ($alpakka$/orientdb/src/test/java/akka/stream/alpakka/orientdb/OrientDBTest.java) { #run-odocument }
+
+
+## Typed messages
+
+Also, it's possible to stream messages which contains any classes. 
+
+Java
+: @@snip ($alpakka$/orientdb/src/test/java/akka/stream/alpakka/orientdb/OrientDBTest.java) { #define-class }
+
+
+Use `OrientDBSource.typed` and `OrientDBSink.typed` to create source and sink instead.
+
+Java
+: @@snip ($alpakka$/orientdb/src/test/java/akka/stream/alpakka/orientdb/OrientDBTest.java) { #run-typed }
+
+
+## Configuration
+
+We can configure the source by `OrientDBSourceSettings`.
+
+Scala (source)
+: @@snip ($alpakka$/orientdb/src/main/scala/akka/stream/alpakka/orientdb/OrientDBSourceSettings.scala) { #source-settings }
+
+| Parameter        | Default | Description                                                                                                              |
+| ---------------- | ------- | ------------------------------------------------------------------------------------------------------------------------ |
+| maxPartitionSize |         | `OrientDBSource` and `OrientDBSink` uses this for initializing DB Connections. |
+| maxPoolSize      |    -1   | `OrientDBSource` and `OrientDBSink` uses this for initializing DB Connections. |
+| skip             |         | `OrientDBSource` uses this property to fetch data from the DB. |
+| limit            |         | `OrientDBSource` uses this property to fetch data from the DB. |
+| dbUrl            |         | url to the OrientDB database. |
+| username         |         | username to connect to OrientDB. |
+| password         |         | password to connect to OrientDB. | 
+
+Also, we can configure the sink by `OrientDBSinkSettings`.

```

Comment:
Is there a better name for the sink-and-flow-settings?

Reply:
Can you please suggest a suitable name?

Reply:
How does `OrientDbUpdateSetting` sound?